include/wl/wl.h

```diff
@@ -60,6 +61,14 @@ class MapNode {
 
 	constexpr std::span<const Adjacency> neighbors() const noexcept { return neighbors_; }
 
+	constexpr bool is_adjacent_to(ID nbor) const noexcept { 
+		auto nbor_itr = find_neighbor(nbor);
+		if (nbor_itr == neighbors_.end()) {
+			return false;
+		}
+		return true;
+	}
```

Comment:
This could be a little simpler: let's do

```c++
constexpr bool is_adjacent_to(ID nbor) const noexcept {
   return find_neighbor(nbor) != neighbors_.end();
}
```

Reply:
(since the `!=` operator returns a `bool` that is functionally equivalent to:
```c++
if (condition) {
   return false;
}
// otherwise
return true;
```
)